src/pages/Environments.js

```diff
@@ -425,19 +420,16 @@ const Environments = () => {
             field: 'project',
             headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
-              const disk = filteredDisks[rowIndex]
-              const galaxyDiskNames = _.map(disk => disk.name, galaxyDisks)
-              const runtimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames), disks)
-              const runtimeDiskNames = _.map(disk => disk.name, runtimeDisks)
-              const multipleRuntimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames) || disk.status === 'Deleting',
-                disksByProject[disk.googleProject]).length > 1
-              const multipleGalaxyDisks = _.remove(disk => _.includes(disk.name, runtimeDiskNames) || disk.status === 'DELETING',
-                disksByProject[disk.googleProject]).length > 1
+              const { status: diskStatus, googleProject } = filteredDisks[rowIndex]
+              const appType = getDiskAppType(filteredDisks[rowIndex])
+              const multipleDisksOfType = _.remove(disk => getDiskAppType(disk) !== appType || disk.status === 'Deleting',
```

Comment:
`diskStatus` instead of `disk.status`

Also, a bit nitty but since it took me some time to parse, I figured I'd mention that it helps me when a constant/function of type boolean has a name that starts with a modal or auxiliary verb, and perhaps even `there` at times.
```suggestion
              const areThereMultipleDisksOfType = _.remove(disk => getDiskAppType(disk) !== appType || diskStatus === 'Deleting',
```

If that's too cumbersome, `isMultipleDisksOfType` would do too even though it's grammatically incorrect.

Reply:
`disk.status` is actually correct instead of `diskStatus`. We are iterating over the disks for the project, and removing any that are Deleting or are of a different "type". `diskStatus` is for the specific disk represented by the row we are rendering.

Honestly, I'm not a fan of either `areThereMultipleDisksOfType` or `isMultipleDisksOfType`. But being a Python person, I'm not used to sticking `get` or `is` in front of everything (which is why I'm now confused!). To me, `multipleDisksOfType` seems clear.

Reply:
> disk.status is actually correct instead of diskStatus. We are iterating over the disks for the project, and removing any that are Deleting or are of a different "type". diskStatus is for the specific disk represented by the row we are rendering.

Ah right - I'd misread it.

> Honestly, I'm not a fan of either areThereMultipleDisksOfType or isMultipleDisksOfType. But being a Python person, I'm not used to sticking get or is in front of everything (which is why I'm now confused!). To me, multipleDisksOfType seems clear.

Especially in absence of static typing where the IDE can help more, clear naming conventions act as type annotation to me. Not a blocker for this PR but I will make a note to discuss offline.